dotnet/README.md

```diff
@@ -1,13 +1,13 @@
 # IBM MQ XMS samples
-The XMS samples are based on the the existing samples shipped with IBM MQ Server and Client packages. These have been tested with .NET Framework 4.6 and Visual Studio Community 2017 v 15.9.5.
+The XMS samples are based on the the existing samples shipped with IBM MQ Server and Client packages. The samples here have been tested with .NET Framework 4.6 and Visual Studio Community 2017 v 15.9.5. Updates have been tested with Visual Studio Community 2019 v 16.1.4 and .NET Framework 4.8.03761.
 
 ## Download
 
-[Windows MQ client v 9.1.1.0 download](https://www-945.ibm.com/support/fixcentral/swg/selectFixes?parent=ibm~WebSphere&product=ibm/WebSphere/WebSphere+MQ&release=9.1.1&platform=Windows+64-bit,+x86&function=fixId&fixids=9.1.1.0-IBM-MQC-Win64+&useReleaseAsTarget=true&includeSupersedes=0)
+[Windows MQ client v 9.1.2.0 download](https://www-945.ibm.com/support/fixcentral/swg/selectFixes?parent=ibm~WebSphere&product=ibm/WebSphere/WebSphere+MQ&release=9.1.2&platform=Windows+64-bit,+x86&function=fixId&fixids=9.1.2.0-IBM-MQC-Win64+&useReleaseAsTarget=true&includeSupersedes=0)
 
 Reference the IBM.XMS library through Visual Studio Solution References option, right click and browse to [MQ default install location](C:\Program Files\IBM\MQ\bin\IBM.XMS.dll).
 
-
+We have included '.sln' and '.csproj' files. These were created with the Visual Studio Community 2019. You'll need to copy the 'env.json' file from the top level directory to '/bin/Debug' in your project directory and adjust to use your own queue manager.
```

Comment:
For the 'env.json' file I recommend adding the file to the project and then change its properties to "Copy if newer".

![image](https://user-images.githubusercontent.com/1583241/61323589-a7393c80-a810-11e9-8ac6-dc79fd457348.png)